### DISCUSSION TITLE
Does Seata support using MariaDB Galera Cluster as store?

### DISCUSSION THREAD

**Discussion by XiaoleiJia1005** [apache/incubator-seata#4325] [Q&A] [answered]
MariaDB Galera Cluster has multiple master servers for w/r. How to configure Seata to connect to all these servers so when one of the server is down, Seata can choose a health one? 

Thanks

**Comment by funky-eyes** [comment, marked as answer]
我个人理解对seata而言应该不需要关注MariaDB Galera Cluster的背后机制,高可用由MariaDB Galera Cluster进行保障即可
